introduction/differences.md

```diff
@@ -29,6 +29,7 @@ The most noteworthy changes since the previous release are described here.
          * Rome-Based HPE DL 385(v1) Gen10
    * Node consoles are now managed by cray-console-node which is based on conman.
```

Comment:
These changes are also needed to this existing line:
- conman -> ConMan
- Remove period

Reply:
I am not going to do this since it is not related to my change.